tests/common/helpers/assertions.py

```diff
@@ -4,3 +4,9 @@ def pytest_assert(condition, message = None):
     __tracebackhide__ = True
     if not condition:
         pytest.fail(message)
+
+def pytest_require(condition, skip_message=""):
+    if not condition:
+        # We can't use pytest.skip here because pytest after 3.0
+        # doesn't allow to call skip outside a test case or fixture.
+        raise pytest.skip.Exception(skip_message, allow_module_level=True)
```

Comment:
allow_module_level=True. Are we assuming that all tests to be run on a specific asic type will reside in a module? We need to call that out in our pytest doc then. 

Also specify the 'allow_module_level' as an arg to the method with a default, so that this method can be reused for other use cases as well in future

Reply:
I checked the pytect doc here: https://docs.pytest.org/en/latest/reference.html#id14
allow_module_level (bool) – Allows this function to be called at module level, skipping the rest of the module. 
The method is reuseable for other cases even allow_module_level is True. When the method is called in a testcase, it will only skip the case. If we call it in module level, all scripts in the module will be skipped.